packages/rspeedy/plugin-react-alias/src/index.ts

```diff
@@ -82,6 +92,7 @@ export function pluginReactAlias(options: Options): RsbuildPlugin {
                   .set('@lynx-js/react/lepus$', reactLepus.mainThread)
                   .set('@lynx-js/react/lepus/jsx-runtime', jsxRuntime.mainThread)
                   .set('@lynx-js/react/lepus/jsx-dev-runtime', jsxDevRuntime.mainThread)
+                 .set('background-only$', backgroundOnly.mainThread)
```

Comment:
I think we already have aliases for `background-only$` at https://github.com/lynx-family/lynx-stack/blob/4f74a12e5664a0bbc9d650a17568be00e62e0865/packages/rspeedy/plugin-react/src/backgroundOnly.ts#L42-L46

Why are we adding it again?